go.mod

```diff
@@ -49,7 +49,7 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/pkg/errors v0.9.1
 	github.com/pmezard/go-difflib v1.0.0
-	github.com/rootless-containers/rootlesskit v0.11.1
+	github.com/rootless-containers/rootlesskit v0.11.2-0.20210121135636-7d41416ded8a
```

Comment:
v0.12.0 is now available

Reply:
thanks, changed the dependency to v0.12.0